inc/modules/export/epub/templates/epub201/opf.php

```diff
@@ -174,10 +174,16 @@
 
 		// First, look if the user has set this themselves.
 		$start_key = false;
-		foreach ( $manifest as $key => $val ) {
-			if ( $val['ID'] > 0 && get_post_meta( $val['ID'], 'pb_ebook_start', true ) ) {
+		$start_id = false;
+		$ebook_options = get_option( 'pressbooks_theme_options_ebook', [] );
+		if ( isset( $ebook_options['ebook_start_point'] ) && ! empty( $ebook_options['ebook_start_point'] ) ) {
+			$start_id = $ebook_options['ebook_start_point'];
+		}
+
+		// Determine the manifest key for the post ID.
+		foreach ( $manifest as $key => $value ) {
+			if ( $start_id === $value['ID'] ) {
 				$start_key = $key;
-				break;
```

Comment:
Any reason that the break was removed? if ` $manifest` has 10,000 items, and `$start_id === $value['ID']` on the third loop, then without the `break` the loop will keep running another 9997 times.